proto/pdpb.proto

```diff
@@ -530,10 +532,14 @@ message ScatterRegionRequest {
     // If group is defined, the regions with the same group would be scattered as a whole group.
     // If not defined, the regions would be scattered in a cluster level.
     string group = 5;
+
+    repeated uint64 regions_id = 6;
```

Comment:
Please add the relationship between it and `region_id`.

Reply:
If there are some fields that need to be deprecated, please refer to the [documentation](https://developers.google.com/protocol-buffers/docs/proto3#options) to mark them.

Reply:
updated.